### PR TITLE
Obada week4 javascript

### PR DESCRIPTION
**Pull request by ObadaElSharbatly**
I am Obada and this is my javascript Homework for week 4

### CODE REVIEW

File: 1-JavaScript/Week4/homework/ex5-wallet/index.js [hunk 1]
```diff
@@ -113,6 +113,6 @@ const quiz = {
       b: 'Joe', 
       c: 'Jane' 
     },
-    answer: undefined,
+    answer: 'c',
```
Comment (Fadi-Rasheed) [inline 1-JavaScript/Week4/homework/ex5-wallet/index.js:116]: Fantastic!

**Review by Fadi-Rasheed — APPROVED** [review]
Very well done!, your code is clear and perfect!

File: 1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js [hunk 2]
```diff
@@ -10,12 +10,8 @@ Let's rewrite it (or _refactor_ it, as experienced developers would call it):
...
 function doubleEvenNumbers(numbers) {
   // TODO rewrite the function body using `map` and `filter`.
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
+  const evenNumbers = numbers.filter((index) => index % 2 === 0);
+  const newNumbers = evenNumbers.map((index) => index * 2);
```
[the remark this answers is not in the document]
Comment (ObadaElSharbatly) [reply 1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js:14]: yeah .. you're right sir I will follow that .. thaaaaaanks ✨

**Comment by ObadaElSharbatly** [comment]
Thank you Mr @Fadi-Rasheed I will follow your advises really thank you for looking carefully in the code ❤✨

**Pull request closed without merging**